src/modules/auth/roles/roles.scss

```diff
@@ -1,18 +1,24 @@
 // wrap all cards in RolesDetailContainer
 .role-detail-wrapper {
-	display: flex;
-	flex-wrap: wrap;
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-template-areas:
+		"a b"
+		"c b"
+		"d d";
+	row-gap: 0.25rem;
+	column-gap: 0.25rem;
 
 	.role-detail-info {
-		width: 50%;
-		display: inline-block;
-		padding-right: 15px;
+		grid-area: a;
 	}
 
 	.role-detail-credentials-area {
-		width: 50%;
-		display: inline-block;
-		padding-left: 15px;
+		grid-area: b;
+		& .data-table-card-body {
+			max-height: 550px;
```

Comment:
@petrKavulok Why it is neccessary to set up max-height here? I thought that grid areas will handle this. Or we can handle that programatically using limit=10 of the datatable

Reply:
I guess it was not necessary in the end. Thank you, good catch!